app/pages/search/SearchPage.js

```diff
@@ -96,7 +106,13 @@ class UnconnectedSearchPage extends Component {
             showMap={showMap}
           />
         )}
+
         <PageWrapper className="app-SearchPage__wrapper" title={t('SearchPage.title')} transparent>
+          <Row>
+            <Col className="app-SearchControlsContainer__control sortControl col-md-offset-8" md={4} sm={6}>
```

Comment:
Dont use bootstrap like this, use offset in input props, not className
the `sortControl` need to follow naming pattern for example `app-SearchControl__sortControl`

Reply:
Corrections made.